webpack.renderer.config.js

```diff
@@ -126,11 +133,39 @@ const rendererConfig = {
     new HtmlWebpackPlugin({
       filename: `index.html`,
       template: `./app/index.ejs`,
-      appModules: process.env.NODE_ENV !== `production` ?
-        path.resolve(__dirname, `app/node_modules`) : false,
+      appModules: production ? false :
+        path.resolve(__dirname, `app/node_modules`),
       styles: fs.readFileSync(`./app/src/renderer/styles/index.css`, `utf8`),
       favicon: `./app/static/icons/favicon.ico`
     }),
+    new CSPWebpackPlugin({
+      'object-src': `'none'`,
+      'base-uri': `'self'`,
+      'default-src': `'self'`,
+      'script-src': [
+        `'self'`,
+        `https://app.appzi.io/`,
+        production ? `https://*.lunie.io` : `https://Localhost:9080`
+      ],
+      'worker-src': `'none'`,
+      // 'style-src': production ? `'self'` : `*`, // SECURITY Appzi is applying styles inline, inquired to them already
+      'style-src': [`'self'`, `'unsafe-inline'`],
+      'connect-src':
+        !production ? `*` : [
+          // third party tools
+          `https://sentry.io`,
+          `https://appzi-collector-b.azurewebsites.net`,
+          `https://keybase.io`,
+          // mainnet
+          `https://stargate.cosmos.network`,
+          `wss://rpc.cosmos.network:26657`,
+          // testnet
+          `https://sntajlxzsg.execute-api.eu-central-1.amazonaws.com/`,
```

Comment:
would be great to have a dns entry for our testnet node (and we should move this over too).